modules/library/metadata/src/main/java/org/geotools/factory/RegistrableFactory.java

```diff
@@ -0,0 +1,10 @@
+package org.geotools.factory;
```

Comment:
header :) New file so I gotta ask

Reply:
Done in e0c8e75. I simply copied some other header - I hope that's ok.